internal/backends/linuxkms/display/egldisplay.rs

```diff
@@ -118,12 +118,27 @@ impl raw_window_handle::HasDisplayHandle for EglDisplay {
 }
 
 pub fn create_egl_display() -> Result<EglDisplay, PlatformError> {
+    let drm_devices = 
+        std::fs::read_dir("/dev/dri/")
+            .map_err(|e| format!("Error opening /dev/dri/: {e}"))?;
+    for device in drm_devices {
+        if let Ok(device) = device.map_err(|e| format!("Error opening DRM device: {e}")) {
```

Comment:
Since the error isn't used I suppose you could also do without :)

```suggestion
        if let Ok(device) = device {
```

Reply:
Coding with my eyes closed. For some reason I thought I was logging it.

Reply:
Is an "Infallible" error anyway, so can safely ignore